assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -27,23 +27,27 @@ const FieldRenderer: React.FC<FieldRendererProps> = (props) => {
 
 	return (
 		<FormControl
-			isInvalid={ Boolean(errorMessage) }
-			isRequired={ required }
-			className={ classNames('form-item', `form-item-${rest.fieldType}`, formControlProps?.className) }
+			isInvalid={Boolean(errorMessage)}
+			isRequired={required}
+			className={classNames(
+				'ee-input__wrapper ee-form-item',
+				`form-item__${rest.fieldType}`,
+				formControlProps?.className
+			)}
 		>
-			<FormLabel htmlFor={ props.input.name }>
-				{ label }
-				{ info && (
-					<Tooltip placement='right' tooltip={ info }>
-						<InfoCircleOutlined />
+			<FormLabel htmlFor={props.input.name}>
+				{label}
+				{info && (
+					<Tooltip placement='right' tooltip={info}>
+						<InfoCircleOutlined className='ee-form-item__help-info' />
```

Comment:
👍

Reply:
it's ok, but maybe we can centralize this with a prop and apply needed styles at higher level,
the intention being a less fragile code

Reply:
would be nice if our icons had a class applied to them by default, ex: `ee-icon--help-info`

Reply:
yes, a generated one based on the component name would be smth interesting

Reply:
done